handler.py

```diff
@@ -19,6 +19,8 @@ def lambda_handler(event, context):
         os.environ['FB_GROUP_ID'],
     )
 
+    sensitive_mailboxes = json.loads(os.environ['SENSITIVE_MAILBOXES'])
```

Comment:
For safety, maybe check if SENSITIVE_MAILBOXES is falsey before trying to parse --- if it's not set we can just set sensitive_mailboxes to an empty dict.

Reply:
nice catch

Reply:
this ok?

```python
sensitive_mailboxes = json.loads(
        os.environ.get(['SENSITIVE_MAILBOXES'], "[]")
    )
```

Reply:
You don't need the brackets around 'SENSITIVE_MAILBOXES'